packages/mongodb-chatbot-server/src/services/mongodbConversations.ts

```diff
@@ -126,6 +127,39 @@ export function makeMongoDbConversationsService(
       }
       return true;
     },
+
+    async commentMessage({
+      conversationId,
+      messageId,
+      comment,
+    }: CommentMessageParams) {
+      const updateResult = await conversationsCollection.updateOne(
+        {
+          _id: conversationId,
+          messages: {
+            $elemMatch: {
+              id: messageId,
+              role: "assistant",
+              rating: { $exists: true },
```

Comment:
as noted here https://github.com/mongodb/chatbot/pull/307/files#r1464163336, my inclination is to not require rating to exist at the ConversationService level